src/client/src/pages/settings/admin/userDetail.tsx

```diff
@@ -62,79 +52,14 @@ export const UserDetail: FC<UserDetailProps> = ({ user, setUser }) => {
 
         // Get the transformed array of unique workgroups with roles
         setUserWorkgroups(getUniqueWorkgroups(user));
+        setIsLoading(false);
```

Comment:
REQ: `setIsLoading(false)` wird unter Umständen, z.B. bei einem unerwarteten Fehler, nicht aufgerufen. Das sollte entsprechend in ein `finally`-Block verschoben werden, mit Error-Handling.